database-mysql/index.js

```diff
@@ -1 +1,33 @@
 // this is where the database connection will happen 
+const Sequelize = require('sequelize');
+
+const host = 'localhost';
+const user = 'root';
+const port = 3306;
+const password = '';
+const database = 'joke-me-off';
```

Comment:
mysql would not allow me to create a database with a dash (-) in the name like joke-me-off